metronome/networking/src/io/iohk/metronome/networking/ConnectionHandler.scala

```diff
@@ -125,12 +161,115 @@ class ConnectionHandler[F[_]: Concurrent, K, M](
     }
   }
 
-  private def callCallBackIfNotClosed(
+  private def handleConflict(
```

Comment:
The calls to `tracers.registered` and `tracers.discarded` are missing from this method now.

Reply:
I forgot to re-add them at some point! thanks for catching it !